frontend/packages/console-shared/src/components/quick-search/QuickSearchList.tsx

```diff
@@ -115,18 +123,21 @@ const QuickSearchList: React.FC<QuickSearchListProps> = ({
           );
         })}
       </DataList>
-      <div className="odc-quick-search-list__all-items-link">
-        {viewAll?.map((catalogLink) => (
-          <Link
-            id={catalogLink.catalogType}
-            to={catalogLink.to}
-            key={catalogLink.catalogType}
-            style={{ fontSize: 'var(--pf-global--FontSize--sm)' }}
-          >
-            {catalogLink.label}
-          </Link>
-        ))}
-      </div>
+
+      {viewAll.length > 0 && (
```

Comment:
This will throw error in case viewAll is null/undefined. 
```suggestion
      {viewAll?.length > 0 && (
```